doc/cascadia/SettingsSchema.md

```diff
@@ -1,185 +1,185 @@
-| -------- | ---- | ----------- | ----------- |
-| `name` | _Required_ | String | Name of the color scheme. |
-| `foreground` | _Required_ | String | Sets the foreground color of the color scheme. |
-| `background` | _Required_ | String | Sets the background color of the color scheme. |
-| `selectionBackground` | Optional | String | Sets the selection background color of the color scheme. |
-| `black` | _Required_ | String | Sets the color used as ANSI black. |
-| `blue` | _Required_ | String | Sets the color used as ANSI blue. |
-| `brightBlack` | _Required_ | String | Sets the color used as ANSI bright black. |
-| `brightBlue` | _Required_ | String | Sets the color used as ANSI bright blue. |
-| `brightCyan` | _Required_ | String | Sets the color used as ANSI bright cyan. |
-| `brightGreen` | _Required_ | String | Sets the color used as ANSI bright green. |
-| `brightPurple` | _Required_ | String | Sets the color used as ANSI bright purple. |
-| `brightRed` | _Required_ | String | Sets the color used as ANSI bright red. |
-| `brightWhite` | _Required_ | String | Sets the color used as ANSI bright white. |
-| `brightYellow` | _Required_ | String | Sets the color used as ANSI bright yellow. |
-| `cyan` | _Required_ | String | Sets the color used as ANSI cyan. |
-| `green` | _Required_ | String | Sets the color used as ANSI green. |
-| `purple` | _Required_ | String | Sets the color used as ANSI purple. |
-| `red` | _Required_ | String | Sets the color used as ANSI red. |
-| `white` | _Required_ | String | Sets the color used as ANSI white. |
-| `yellow` | _Required_ | String | Sets the color used as ANSI yellow. |
-
-## Keybindings
-Properties listed below are specific to each custom key binding.
-
-| Property | Necessity | Type | Description |
-| -------- | ---- | ----------- | ----------- |
-| `command` | _Required_ | String | The command executed when the associated key bindings are pressed. |
-| `keys` | _Required_ | Array[String] | Defines the key combinations used to call the command. |
-
-### Implemented Keybindings
-
-Bindings listed below are per the implementation in `src/cascadia/TerminalApp/AppKeyBindingsSerialization.cpp`
-
-- copy
-- copyTextWithoutNewlines
-- paste
-- newTab
-- openNewTabDropdown
-- duplicateTab
-- newTabProfile0
-- newTabProfile1
-- newTabProfile2
-- newTabProfile3
-- newTabProfile4
-- newTabProfile5
-- newTabProfile6
-- newTabProfile7
-- newTabProfile8
-- closeWindow
-- closeTab
-- closePane
-- switchToTab
-- nextTab
-- prevTab
-- increaseFontSize
-- decreaseFontSize
-- resetFontSize
-- scrollUp
-- scrollDown
-- scrollUpPage
-- scrollDownPage
-- switchToTab0
-- switchToTab1
-- switchToTab2
-- switchToTab3
-- switchToTab4
-- switchToTab5
-- switchToTab6
-- switchToTab7
-- switchToTab8
-- openSettings
-- splitHorizontal
-- splitVertical
-- resizePaneLeft
-- resizePaneRight
-- resizePaneUp
-- resizePaneDown
-- moveFocusLeft
-- moveFocusRight
-- moveFocusUp
-- moveFocusDown
+# Profiles.json Documentation

```

Comment:
it looks like this commit changed the line endings on the Settings Schema document. Is it possible for you to resolve this? It makes it very difficult to review the changes (and may cause issues for future editors)